.travis.yml

```diff
@@ -1,8 +1,21 @@
 language: ruby
+sudo: false
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
-  - 2.3.0
+  - 2.2.6
+  - 2.3.3
+  - 2.4.0
 before_install:
+  - gem update --system
```

Comment:
Well, I guess we don't absolutely need it, but Travis sometimes has an older version of Rubygems installed and this brings it up to the latest version.
http://take.ms/PZaZd